### PR TITLE
Fixes on Layout component

### PR DESCRIPTION
**Pull request by stsrki**
I have added some fixes:

- When first layout is initialized it will add `.b-body-layout` to `body` element
- Moved all strongly typed classnames to ClassProvider
- Line height on `.layout-header` was messing with Bar
- Refactored bootstrap scss files

@MitchellNZ

### CODE REVIEW

File: Source/Blazorise/Styles/_layout.scss [hunk 1]
```diff
@@ -30,8 +30,8 @@
     /*padding: 0 50px;*/
     color: rgba(0, 0, 0, 0.65);
     /*background: #001529;*/
-    line-height: 64px;
-    &-fixed {
+    /*line-height: 64px;*/
+    /*&-fixed {
```
Comment (MitchellNZ) [inline Source/Blazorise/Styles/_layout.scss:34]: @stsrki did you mean to comment this out?
Without this, the `fixed` header will not work.

Comment (stsrki) [reply Source/Blazorise/Styles/_layout.scss:34]: Yes, I moved that to ant design css. It was messing with bootstrap and bulma.

Comment (MitchellNZ) [reply Source/Blazorise/Styles/_layout.scss:34]: Oh I see! I added it back in my PR - wasn't working for me without it in Ant Design and Material.. I will test again.

Sorry about that!

Comment (stsrki) [reply Source/Blazorise/Styles/_layout.scss:34]: I'm just merging PR. I don't see any problem. Do I need to wait a little!

Comment (MitchellNZ) [reply Source/Blazorise/Styles/_layout.scss:34]: If you look at my changes to this file in [my latest PR](https://github.com/stsrki/Blazorise/pull/713/files), you'll see I uncommented this.

I've been testing and I can't see any problems its causing to Boostrap or Bulma? Could you please explain some more and I can look into a fix.

Also, I don't see where you moved this too in Ant Design CSS?

Comment (MitchellNZ) [reply Source/Blazorise/Styles/_layout.scss:34]: Just to be clear, I am talking about the comment to line `34`:

` /*&-fixed {`

Not the `line-height`, I see the issue with that now!

Comment (stsrki) [reply Source/Blazorise/Styles/_layout.scss:34]: Yeah, the `line-height` was making problem. I merged everything to dev090 yesterday. i'm not sure if you have synced with latest changes.

Comment (MitchellNZ) [reply Source/Blazorise/Styles/_layout.scss:34]: Yes, I see that. No issues with moving the `line-height`!
I commented on my PR #713 with exactly what I am referring to

**Pull request merged**